src/hdmf/build/objectmapper.py

```diff
@@ -716,6 +716,7 @@ def __add_attributes(self, builder, attributes, container, build_manager, source
                         msg = "invalid type for reference '%s' (%s) - "\
                               "must be AbstractContainer" % (spec.name, type(attr_value))
                     raise ValueError(msg)
+                breakpoint()
```

Comment:
breakpoint should be removed

Reply:
Sorry, this code doesn't actually work and needs more debugging, hence the WIP and draft PR status. I just pushed it so that you and @ajtritt could take a look at it with me when we meet.